app/components/Exchange/Exchange.jsx

```diff
@@ -316,23 +316,21 @@ class Exchange extends React.Component {
         return this.state.feeStatus[asset.get("id")] && this.state.feeStatus[asset.get("id")].fee;
     }
 
-    _verifyFee(fee, sellAmount, sellBalance, coreBalance) {
+    _verifyFee(fee, sell, sellBalance, coreBalance) {
         let coreFee = this._getFee();
 
-        let sellSum = fee.getAmount() + sellAmount;
-        if (fee.asset_id === "1.3.0") {
+        if (fee.asset_id === "1.3.0") { // Very strange check
             if (coreFee.getAmount() <= coreBalance) {
                 return "1.3.0";
             } else {
                 return null;
             }
         } else {
+            let sellSum = (sell.asset_id == fee.asset_id) ? (fee.getAmount() + sell.getAmount()) : sell.getAmount();
             if (sellSum <= sellBalance) { // Sufficient balance in asset to pay fee
                 return fee.asset_id;
-            } else if (coreFee.getAmount() <= coreBalance && fee.asset_id !== "1.3.0") { // Sufficient balance in core asset to pay fee
```

Comment:
Why remove the fallback to core fees? It might not be necessary anymore due to better initial checking of what assets are available to pay fees, but I prefer to keep it.

Reply:
that is the main problem of #803 : if i, for example choose bitCNY to pay fee from, it better to say that there is not enough  amount than changing fee asset without any notification.

Reply:
The bitCNY option is only visible in the first place if you actually have the bitCNY to pay it with. This issue only occurs when you're trying to sell your whole stash.

Reply:
yeah, that's true. ok, i'll put it back

Reply:
And I prefer to fallback to BTS anyway instead of issuing error messages since that is better (cheaper) for the user anyway